classes/core-image-pelux-qtauto.bbclass

```diff
@@ -0,0 +1,21 @@
+#
+#   Copyright (C) 2017 Pelagicore AB
+#   SPDX-License-Identifier: MIT
+#
+
+DESCRIPTION = "Class for inheriting a Pelux image with Qt frontend"
+
+inherit core-image-pelux
+
+# A bunch of QtAuto components
+QTAUTO_COMPONENTS := " \
+    qtapplicationmanager \
+    qtivi \
+    qtvirtualkeyboard \
+    dbus-session \
```

Comment:
Maybe this should not be stated here explicitly but pulled in as a part of `qtappman.bbappend` dependency.

Reply:
You mean only dbus-session, right?

Reply:
That's correct, sorry for not being explicit.